.github/workflows/pythonapp.yml

```diff
@@ -43,11 +44,32 @@ jobs:
       - name: Test
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        run: invoke test --coverage
+        run: |
+          invoke test --coverage
```

Comment:
Missed replacing the `python -m pip install` for this step?

Reply:
I'll remove the newline. Can't replace python -m pip install b/c of windows